### PR TITLE
Sync 2.7x-support branch with latest changes

### PR DESCRIPTION
**Pull request by sercero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +45,43 @@ If you are upgrading from a previous version of blender2ogre, and having problem
...
+You can check the arguments passed to `OgreMeshTool.exe` in the Blender console. (`Window Menu > Toggle System Console`)
+
+Blender will export the material format in a Ogre (V1) format. This is not compatible with OgreNext (V2.*). You should manually convert them to a material.json file. See the [Ogre Wiki: HLMS Materials](https://wiki.ogre3d.org/HLMS+Materials) for more information.
+
+## Importing Meshes
+As of `blender2ogre` version *0.8.2*, the Kenshi Importer has been integrated into `blender2ogre` with the following features:
  - Import mesh from `.xml` as well as `.mesh` files
  - Option to be able to merge imported submeshes or keep them separate
- - Parsing/Conversion of materials into Blender (not great, but most matching properties have been considered)
+ - Parsing/Conversion of materials into Blender (just create a Principled BSDF material and add a texture with the proper UV mapping)
```
Comment (paroj) [inline README.md:84]: I would suggest to leave the README as is. "Principled BSDF" is a Blender 2.8+ only thing

File: README.md [hunk 2]
```diff
@@ -72,7 +104,7 @@ Setting any value other than the default `(0, 0, 0)` will result in a mesh with
 As of OGRE 1.13 a new feature has been added to the DotScene Plugin where it now accepts the static / instanced keywords for entities.
 (for more information read the [DotScene Plugin README](https://github.com/OGRECave/ogre/blob/master/PlugIns/DotScene/README.md)).
 
-To use this feature create a new group (Ctrl+G) named as `static.<Group Name>` or `instanced.<Instance Manager Name>` and blender2ogre will automatically add the corresponding attribute to the exported entities in the Scene.
```
Comment (paroj) [inline README.md:75]: collections are blender 2.8 only too

File: README.md [hunk 3]
```diff
@@ -44,11 +45,43 @@ If you are upgrading from a previous version of blender2ogre, and having problem
...
+    - Else Tangents: "none"
+  - Optimise Vertex buffers for shaders: ticked
+  - Vertex buffer options: **puqs**
+
+You can check the arguments passed to `OgreMeshTool.exe` in the Blender console. (`Window Menu > Toggle System Console`)
+
+Blender will export the material format in a Ogre (V1) format. This is not compatible with OgreNext (V2.*). You should manually convert them to a material.json file. See the [Ogre Wiki: HLMS Materials](https://wiki.ogre3d.org/HLMS+Materials) for more information.
+
+## Importing Meshes
+As of `blender2ogre` version *0.8.2*, the Kenshi Importer has been integrated into `blender2ogre` with the following features:
```
Comment (paroj) [inline README.md:81]: this will be shown on the 0.7 branch, so we should refer to 0.7.4

File: README.md (hunk 1 above)
Comment (paroj) [reply README.md:84]: also the screenshots are all from blender 2.8, which probably confuse more than they help

**Comment by paroj** [comment]
as there is no CI for 2.7 to verify, I assume you tested it on 2.7?

**Comment by sercero** [comment]
> as there is no CI for 2.7 to verify, I assume you tested it on 2.7?
Yes.

Regarding the README, you are right some things don't apply but it has to be updated anyways.

Will look into that. Thanks!

**Comment by sercero** [comment]
I resolved the issues.

**Pull request merged**